docs/example/README.md

```diff
@@ -0,0 +1,16 @@
+# Baseline reference configuration
+
+This repo provides a simple example reference configuration for the `cluster-compare` plugin.
+
+## Goal
+
+Create a minimal baseline reference configuration that works with `oc cluster-compare` on any OpenShift cluster. This helps users get started with the cluster-compare plugin quickly and easily.
+
+## Usage
+
+Run the reference configuration against your cluster:
+
+````
+oc cluster-compare -r https://raw.githubusercontent.com/openshift/kube-compare/main/docs/example/metadata.yaml
+````
+
```

Comment:
I think it's complaining about this blank line at the end, too.

Reply:
Updated

This is a very minimal example too. I'm sure it could be built on with more example errors etc.